perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -67,7 +69,8 @@ void RoiClusterFusionNode::postprocess(DetectedObjectsWithFeature & output_clust
   for (auto & feature_object : output_cluster_msg.feature_objects) {
     if (
       feature_object.object.classification.front().label !=
-      autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) {
+        autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN ||
+      feature_object.object.existence_probability > min_roi_existence_prob_) {
```

Comment:
I think `feature_object.object.existence_probability` is always zero  because it's from clustering algorithm, so `feature_object.object.existence_probability > min_roi_existence_prob_` is false. When this probability is updated to non-zero value?

Reply:
@yukke42 Thanks for your review.  Here I forced updating existence_probability[ here ](https://github.com/badai-nguyen/autoware.universe/blob/feature/fuse_unknown_roi/perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp#L208-L213) to differentiate UKNOWN objects of roi_cluster_fusion result which detected by camera as UKNOWN ROI. Do you think it is good idea?

Reply:
Sorry, I missed that this line is in the postprocess function. I think it's fine.

Reply:
I think using the same rlue for all classes is less confusion.